applications/h264/h264_endoscopy_tool_tracking/python/h264_endoscopy_tool_tracking.py

```diff
@@ -0,0 +1,362 @@
+
+        request_condition = AsynchronousCondition(self, "request_condition")
+        video_decoder_request = VideoDecoderRequestOp(
+            self,
+            name="video_decoder_request",
+            async_scheduling_term=request_condition,
+            videodecoder_context=video_decoder_context,
+            **self.kwargs("video_decoder_request"),
+        )
+
+        video_decoder_response = VideoDecoderResponseOp(
+            self,
+            name="video_decoder_response",
+            pool=BlockMemoryPool(
+                self,
+                name="pool",
+                storage_type=MemoryStorageType.DEVICE,
+                block_size=source_block_size,
+                num_blocks=source_num_blocks,
+            ),
+            videodecoder_context=video_decoder_context,
+            **self.kwargs("video_decoder_response"),
+        )
+
+        decoder_output_format_converter = FormatConverterOp(
+            self,
+            name="decoder_output_format_converter",
+            pool=BlockMemoryPool(
+                self,
+                name="pool",
+                storage_type=MemoryStorageType.DEVICE,
+                block_size=source_block_size,
+                num_blocks=source_num_blocks,
+            ),
+            **self.kwargs("decoder_output_format_converter"),
+        )
+
+        rgb_float_format_converter = FormatConverterOp(
+            self,
+            name="rgb_float_format_converter",
+            pool=BlockMemoryPool(
+                self,
+                name="pool",
+                storage_type=MemoryStorageType.DEVICE,
+                block_size=source_block_size,
+                num_blocks=source_num_blocks,
+            ),
+            **self.kwargs("rgb_float_format_converter"),
+        )
+
+        model_file_path = self.sample_data_path + "/tool_loc_convlstm.onnx"
+        engine_cache_dir = self.sample_data_path + "/engines"
+
+        lstm_inferer = LSTMTensorRTInferenceOp(
+            self,
+            name="lstm_inferer",
+            model_file_path=model_file_path,
+            engine_cache_dir=engine_cache_dir,
+            pool=UnboundedAllocator(self, name="pool"),
+            cuda_stream_pool=CudaStreamPool(
+                self,
+                name="cuda_stream",
+                dev_id=0,
+                stream_flags=0,
+                stream_priority=0,
+                reserved_size=1,
+                max_size=5,
+            ),
+            **self.kwargs("lstm_inference"),
+        )
+
+        tool_tracking_postprocessor = ToolTrackingPostprocessorOp(
+            self,
+            name="tool_tracking_postprocessor",
+            device_allocator=UnboundedAllocator(self, "device_allocator"),
+            host_allocator=UnboundedAllocator(self, "host_allocator"),
+            **self.kwargs("tool_tracking_postprocessor"),
+        )
+
+        record_output = self.from_config("record_output").__bool__
```

Comment:
I don't think you want the __bool__ method? The following should cast to bool for a parameter set to true or false in the YAML.
```suggestion
        record_output = bool(self.from_config("record_output"))
```